src/blog/2023/10/community-news-10.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Community News October 2023
+subtitle: Your monthly update for the FlowFuse and Node-RED communities
+description: News from the FlowFuse and Node-RED communities
+date: 2023-10-05
+authors: ["ian-skerrett"]
+image: "/images/blog/community-news.jpeg"
+tags:
+    - posts
+    - newsletter
+---
+
+Welcome to the FlowFuse newsletter for October 2023, a monthly roundup of what’s been happening with FlowFuse and the wider Node-RED community. 
+
+<!--more-->
+
+## Quicker Releases for FlowFuse Cloud and New Changelog
+
+FlowFuse is committed to delivery new features to our customers and community as fast as possible. We are now rolling out new builds of FlowFuse to FlowFuse Cloud more regularly than the previous monthly release. This will allow FlowFuse Cloud users to benefit from the latest enhancements as they become available. The self-hosted version of FlowFuse will continue to have releases every 4 weeks.
+
+To allow FlowFuse Cloud users to keep up to date on the new changes, we have started a [Changelog](/changelog/) to announce newly deployed features. 
+
+
+## Recent Changelog Updates
+
+- [Private Node Support](/changelog/2023/09/custom-node-support/)
+- [DevOps Pipeline with action selection](/changelog/2023/09/devops-actions/)
+- [Usability improvements to Device Management](/changelog/2023/09/snapshots-devices/)
+- [Introducng the Enterprise tier](/changelog/2023/09/introduction-enterprise-tier/)
+- [API Endpoints for DevOps Pipeline](/changelog/2023/09/pipeline-api/)
+
+## Upcoming events
+
+### Dashboard 2.0 - Where we are, and what’s next?
+
+The next generation of the Node-RED dashboard is starting to mature. Development work on Dashboard 2.0 started earlier this year and amazing progress has been made already. Join Joe Pavitt, leader of the Dashboard 2.0 project, who will demonstrate the new dashboard project and discuss future plans during our October webinar.
+
+[Sign-up today](/webinars/2023/dashboard-20/) to join us on October 26. 
+
+
+## From our Blog
+
+* [How to use private custom nodes in FlowFuse](/blog/2023/10/use-private-custom-nodes-with-flowfuse/) - a new FlowFuse 1.12 feature allows you to use custom nodes.
```

Comment:
This is the same item as `Private Node support` earlier, I suggest using this link above, and removing this bullet point